CONTRIBUTING.md

```diff
@@ -37,12 +41,12 @@ Persons contributing to this project agree to the [code of conduct](./CONDUCT.md
    - If you receive feedback, make changes on your desktop and push to your branch on GitHub: the pull request will update automatically.
    - **If a major change is made by any person, tag other two. If this change is approved by all → accept pull request.
 
-##How to Contribute
+## How to Contribute
 
-####Code comment 
+#### Code comment 
 - Functions need to be documented clearly.
 
-####Commit message
+#### Commit message
 - Big changes:
```

Comment:
Small changes:
```
git commit -m "Concise summary of what you did here"
```

Major changes:
```
git commit -m "Title of commit message
Details of your changes"
```

Reply:
We talk about it in the `How to contributing` part.